python/setup.py

```diff
@@ -5,14 +5,29 @@ def load_dependencies():
         required = f.read().splitlines()
     return required
 
+
+def getReadMe():
+    with open("README.md", "r") as f:
+        long_description = f.read()
+    return long_description
+
 setup(
     name="ml4ir",
     packages=find_namespace_packages(include=["ml4ir.*"]),
     version="0.0.1",
     description="Machine Learning libraries for Information Retrieval",
+    long_description=getReadMe(),
+    long_description_content_type="text/markdown",
     author="Search Relevance, Salesforce",
-    author_email="searchrelevancyscrumteam@salesforce.com ",
-    data_files=[("ml4ir/build", ["ml4ir/build/Dockerfile"])],
+    author_email="searchrelevancyscrumteam@salesforce.com",
+    url="https://www.salesforce.com/",
+    classifiers=[
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3 :: Only"
+    ],
+    include_package_data=True,
+    data_files=[("ml4ir/build", ["ml4ir/build/Dockerfile", "ml4ir/build/run_driver.sh"])],
```

Comment:
should we add the `docker-compose.yml` here? Or doing `pip install` is mutually exclusive with `docher-compose up`. In general, after doing `pip install` I am not able to run the scripts shown in the remaining of the README. Which makes sense, because the paths indicated there are obsolete. We need to fix this I guess. Disclaimer: I am a docker beginner, I need to learn more about it.

Reply:
From what I understand, to use docker we need to git clone the repo and not pip install. @lastmansleeping might have more insights into this

Reply:
I don't think there is a need to add `docker-compose.yml` to setup. We can also pull the `Dockerfile` now from the virtualenv/pythonpath because we have made sure to add it to the pip installable ml4ir package. So it should be good.

Reply:
ok, thanks!